README.md

```diff
@@ -212,7 +212,7 @@ You can set the `SIZE` and `PVC` values aswell
 make MODEL_PATH="PATH_TO_A_FILE" NAME=my-model SIZE=1G PVC=my-new-PVC create
 ```
 
-In `pipelines/tekton/build-container-image-pipeline` you can set the
+In `pipelines/tekton/obsolete/build-container-image-pipeline` you can set the
 `fetch-model` to `pvc` for the pipeline to copy the file from the
 `model-workspace` which can be set to the PVC created in the last step
```

Comment:
@LaVLaS Support for copying the model from a PVC to a workspace wasn't migrated to the e2e pipeline. Is it on purpose? If so, I would propose on removing this part from docs.

Reply:
Follow-up from an offline conversation: If this is the only reference to `pipelines/tekton/obsolete` in the repo after this change then I think we can safely delete the `pipelines/tekton/obsolete` directory from the repo.

After that this is an LGTM from me

Reply:
@LaVLaS I originally thought you wanted to "archive" the old pipelines for some reason, but it's true it was a [subtask](https://issues.redhat.com/browse/RHOAIENG-1881) of the [JIRA](https://issues.redhat.com/browse/RHOAIENG-1578) you closed. Regarding the reference to `pipelines/tekton/obsolete` it's purely on us whether we want to retain this reference in the docs or not. If we want to remove the pipelines completely, I have no issues in doing that, I will just add one more commit on top of this so we have the pipelines archived with the correct documentation in the git history.

Reply:
> I will just add one more commit on top of this so we have the pipelines archived with the correct documentation in the git history.

That sounds good. Apologies for the confusion.